tests/readme.rst

```diff
@@ -0,0 +1,42 @@
+==================
+OpenMOC Test Suite
+==================
+
+The purpose of this test suite is to ensure that that all user input options
+can be used successfully without breaking the code. The test suite is based on
+regression or integrated testing where different types of input files are
+configured and the full OpenMOC code is executed. Results from simulations
+are compared with expected results. The test suite is comprised of many tests
+which reside in sub-directories in the tests directory.
+
+The test suite is designed to integrate with cmake using ctest_. To run the
+full test suite run:
+
+.. code-block:: sh
+
+    python run_tests.py
+
+A subset of build configurations and/or tests can be run. To see how to use
+the script run:
+
+.. code-block:: sh
+
+    python run_tests.py --help
+
+As an example, say we want to run all tests with debug flags only on tests
+that have cone and plot in their name. Also, we would like to run this on
+4 processors. We can run:
```

Comment:
nope, this was from the README I pulled for OpenMC's test suite. I'll update it accordingly.